boards/arm/stm32g071b_disco/stm32g071b_disco.dts

```diff
@@ -177,6 +177,8 @@
 	 */
 	psc-ucpdclk = <1>;
 	hbitclkdiv = <27>;
+	pinctrl-0 = <&ucpd1_cc1_pa8 &ucpd1_cc2_pb15>;
```

Comment:
... since the property is made required before it is defined in these boards.

Reply:
I changed the commit order

Reply:
Sorry, I didn't fully understand what you meant.

The following PR must merge before I can use the properties: https://github.com/zephyrproject-rtos/hal_stm32/pull/144/commits